src/main/cpp/src/query_operations/broad_combined_gvcf.cc

```diff
@@ -796,8 +796,7 @@ void BroadCombinedGVCFOperator::operate(Variant& variant, const VariantQueryConf
   if (m_query_config->is_defined_query_idx_for_known_field_enum(GVCF_ID_IDX)) {
     auto ID_query_idx = m_query_config->get_query_idx_for_known_field_enum(GVCF_ID_IDX);
     merge_ID_field(variant, ID_query_idx);
-    if (!m_ID_value.empty())
-      bcf_update_id(m_vcf_hdr, m_bcf_out, m_ID_value.c_str());
+    bcf_update_id(m_vcf_hdr, m_bcf_out, m_ID_value.empty() ? "." : m_ID_value.c_str());
```

Comment:
Strange that earlier PRs didn't catch this - again functionally correct, byte-wise different.

Reply:
Wonder if some earlier changes have introduced a subtle change in behavior?